account_consolidation/wizard/consolidation_check_view.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_consolidation_check_form" model="ir.ui.view">
+        <field name="name">account.consolidation.check.form</field>
+        <field name="model">account.consolidation.check</field>
+        <field name="type">form</field>
+        <field name="arch" type="xml">
+            <form string="Consolidation: Checks">
+                <field name="state" invisible="True" />
+                <div states="open">
+                    <label string="Prepare your consolidation and make sure that your consolidation will be correct by proceeding with the checks." />
```

Comment:
split long texts on multiple lines. Here you can use `<strong />` and keep the original field's label